src/components/Tabs/README.md

```diff
@@ -25,33 +184,131 @@
 | wrapTo           | `wrapTo?(item: TabsItemProps, node: React.ReactNode, index: number): void` |          |              | Allows to wrap `TabItem` into another component or render custom tab |
 | className        | `String`                                                                   |          |              | CSS-class of element                                                 |
 
-### Examples
+## Tabs.Item
 
-```jsx harmony
-<Tabs
-  items={[
-    {
-      id: 'firstTab',
-    },
-    {
-      id: 'secondTab',
-      title: 'Second Tab',
-      meta: 'Second tab meta data',
-      hint: 'The Second One',
-    },
-    {
-      id: 'thirdTab',
-      title: 'Third Tab',
-      meta: 'Third tab meta data',
-    },
-    {
-      id: 'disabledTab',
-      title: 'This tab is disabled',
-      meta: 'Disabled tab meta data',
-      disabled: true,
-    },
-  ]}
-  activeTab={activeTab}
-  onSelectTab={(tabId) => setActiveTab(tabId)}
-/>
+Used for render tabs item.
+
+### Icon
+
+Used if you need to display an icon for a tabs item.
+
+<!--LANDING_BLOCK
+<ExampleBlock
+    code={`
+<Tabs activeTab="first">
+    <Tabs.Item
+        icon={<Icon size={16} data={GearIcon} />}
+        id="first"
+        title="Tab with icon"
+        onClick={() => {}}
+    />
+    <Tabs.Item id="second" title="Tab without icon" onClick={() => {}} />
+</Tabs>
+`}
+>
+    <UIKit.Tabs activeTab="first">
+        <UIKit.Tabs.Item
+            icon={
+                <UIKit.Icon data={() => (
+                    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="16" height="16" class="yc-icon" fill="currentColor" stroke="none" aria-hidden="true"><svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16"><path fill="currentColor" fill-rule="evenodd" d="M7.199 2H8.8a.2.2 0 0 1 .2.2c0 1.808 1.958 2.939 3.524 2.034a.199.199 0 0 1 .271.073l.802 1.388a.199.199 0 0 1-.073.272c-1.566.904-1.566 3.164 0 4.069a.199.199 0 0 1 .073.271l-.802 1.388a.199.199 0 0 1-.271.073C10.958 10.863 9 11.993 9 13.8a.2.2 0 0 1-.199.2H7.2a.199.199 0 0 1-.2-.2c0-1.808-1.958-2.938-3.524-2.034a.199.199 0 0 1-.272-.073l-.8-1.388a.199.199 0 0 1 .072-.271c1.566-.905 1.566-3.165 0-4.07a.199.199 0 0 1-.073-.271l.801-1.388a.199.199 0 0 1 .272-.073C5.042 5.138 7 4.007 7 2.2c0-.11.089-.199.199-.199ZM5.5 2.2c0-.94.76-1.7 1.699-1.7H8.8c.94 0 1.7.76 1.7 1.7a.85.85 0 0 0 1.274.735 1.699 1.699 0 0 1 2.32.622l.802 1.388c.469.813.19 1.851-.622 2.32a.85.85 0 0 0 0 1.472 1.7 1.7 0 0 1 .622 2.32l-.802 1.388a1.699 1.699 0 0 1-2.32.622.85.85 0 0 0-1.274.735c0 .939-.76 1.7-1.699 1.7H7.2a1.7 1.7 0 0 1-1.699-1.7.85.85 0 0 0-1.274-.735 1.698 1.698 0 0 1-2.32-.622l-.802-1.388a1.699 1.699 0 0 1 .622-2.32.85.85 0 0 0 0-1.471 1.699 1.699 0 0 1-.622-2.321l.801-1.388a1.699 1.699 0 0 1 2.32-.622A.85.85 0 0 0 5.5 2.2Zm4 5.8a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0ZM11 8a3 3 0 1 1-6 0 3 3 0 0 1 6 0Z" clip-rule="evenodd"></path></svg></svg>
```

Comment:
There should be jsx, not raw SVG. Also extra svg wrapper can be omitted.

Reply:
Menu is not a reference for this)

Reply:
Ok, if we have reference for this? In button component is similarly

Reply:
Button has correct SVG examples

Reply:
Paste from button)